src/Common/src/Interop/Unix/System.Native/Interop.Poll.cs

```diff
@@ -6,38 +6,36 @@
 
 internal static partial class Interop
 {
-    internal static partial class libc
+    internal static partial class Sys
     {
-        [Flags]
         internal enum PollFlags : short
         {
-            POLLIN      = 0x0001,       /* any readable data available */
-            POLLOUT     = 0x0004,       /* data can be written without blocking */
-            POLLERR     = 0x0008,       /* some poll error occurred */
-            POLLHUP     = 0x0010,       /* file descriptor was "hung up" */
-            POLLNVAL    = 0x0020,       /* requested events "invalid" */
+            POLLIN   = 0x0001,  /* any readable data available */
+            POLLOUT  = 0x0004,  /* data can be written without blocked */
+            POLLERR  = 0x0008,  /* an error occurred */
+            POLLHUP  = 0x0010,  /* the file descriptor hung up */
+            POLLNVAL = 0x0020,  /* the requested events were invalid */
         }
 
-        [StructLayout(LayoutKind.Sequential)]
-        internal struct pollfd
+        internal struct PollFd
```

Comment:
Nit: PollFD because FD is an acronym not an abbreviation. (Most confusing guideline of all time, but that's what we have in Framework Design Guidelines.)